mbed_lstools/lstools_linux_generic.py

```diff
@@ -111,7 +114,12 @@ def list_mbeds(self):
                     mbed['target_id'] = mbed_htm_target_id
             mbed['target_id_usb_id'] = tid
             mbed['target_id_mbed_htm'] = mbed_htm_target_id
-            result.append(mbed)
+            if mbed['mount_point'] != None or self.list_unmounted:
+                result.append(mbed)
+            else:
+                logging.warn(
+                    "MBED with target id '%s' is connected, but not mounted. "
+                    "Use the '-u' flag to include it in the list." % mbed['target_id'])
```

Comment:
The whole code base currently isn't using the `logging` module. This should probably change in the future, but in the meantime I'd suggest using the `debug` function defined here: https://github.com/ARMmbed/mbed-ls/blob/408cd77254b6d476b730a18c1c142f5614b9be85/mbed_lstools/lstools_base.py#L545

You can see how it's used here: https://github.com/ARMmbed/mbed-ls/blob/92a85fd4aee17949da583fa61eff8c3bca95b9c0/mbed_lstools/lstools_linux_generic.py#L151

Reply:
👍 I'll submit another PR for use of logging.